integration-tests/contract-multisig-transfer-implicit.spec.ts

```diff
@@ -4,14 +4,13 @@ import { genericMultisig } from "./data/multisig";
 
 CONFIGS().forEach(({ lib, rpc, setup, createAddress }) => {
   const Tezos = lib;
-  const test = require('jest-retries')
 
   describe(`Generic Multisig: ${rpc}`, () => {
     beforeEach(async (done) => {
       await setup()
       done()
     })
-    test('test manager transfers scenarios', 2, async (done: () => void) => {
+    test('test manager transfers scenarios', async (done: () => void) => {
```

Comment:
This part can be removed as well: `: () => void`

Reply:
removed